flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStValueState.java

```diff
@@ -95,4 +100,32 @@ public V deserializeValue(byte[] valueBytes) throws IOException {
         inputView.setBuffer(valueBytes);
         return getValueSerializer().deserialize(inputView);
     }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public ForStDBGetRequest<ContextKey<K>, V> buildDBGetRequest(
+            StateRequest<?, ?, ?> stateRequest) {
+        Preconditions.checkArgument(stateRequest.getRequestType() == StateRequestType.VALUE_GET);
+        ContextKey<K> contextKey =
+                new ContextKey<>((RecordContext<K>) stateRequest.getRecordContext());
+        return ForStDBGetRequest.of(
+                contextKey, this, (InternalStateFuture<V>) stateRequest.getFuture());
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public ForStDBPutRequest<ContextKey<K>, V> buildDBPutRequest(
+            StateRequest<?, ?, ?> stateRequest) {
+        Preconditions.checkArgument(
+                stateRequest.getRequestType() == StateRequestType.VALUE_UPDATE
+                        || stateRequest.getRequestType() == StateRequestType.CLEAR);
+        ContextKey<K> contextKey =
+                new ContextKey<>((RecordContext<K>) stateRequest.getRecordContext());
+        V value =
+                (stateRequest.getRequestType() == StateRequestType.CLEAR)
+                        ? null  // "Delete(key)" is equivalent to "Put(key, null)"
+                        : (V) stateRequest.getPayload();
```

Comment:
What's the content in `stateRequest.getPayload()`, I remember it was the serialized key in [last PR](https://github.com/apache/flink/pull/24681#discussion_r1576167215).